clang/test/SemaSYCL/unsupported_math.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clangxx --sycl %s -c -o a_kernel.bc -Xclang -verify
+
+#include <CL/sycl.hpp>
```

Comment:
Please, do not include SYCL headers to clang's tests.

Reply:
Removed the headers. Thanks